app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
```

Comment:
It's better to use parentheses when importing multiple classes/modules from the same file. This makes the code more readable. Here is how you can do it:

python
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```